doc/en/components/charts/chart-features-navigation.md

```diff
@@ -0,0 +1,216 @@
+    dragModifier="Shift"
+    panModifier="Alt"
+    isHorizontalZoomEnabled="true"
+    isVerticalZoomEnabled="true">
+</igx-data-chart>
+```
+
+```tsx
+<IgrDataChart dataSource={this.data}
+    width="100%"
+    height="400px"
+    defaultInteraction="None"
+    dragModifier="Shift"
+    panModifier="Alt"
+    isHorizontalZoomEnabled={true}
+    isVerticalZoomEnabled={true}/>
+```
+
+```html
+<igc-data-chart id="chart"
+    width="100%"
+    height="400px"
+    default-interaction="None"
+    drag-modifier="Shift"
+    pan-modifier="Alt"
+    is-horizontal-zoom-enabled="true"
+    is-vertical-zoom-enabled="true">
+</igc-data-chart>
+```
+
+<!-- ## Chart Navigation with Overview Plus Detail Pane
+
+In the `XamDataChart` control, there is an overlaid control that allows navigation. This control supports a preview of most supported series types as well as all navigation methods described above. This overlay is the overview plus detail pane, and it can be enabled by setting the `overviewPlusDetailPaneVisibility` property.
+
+The following is a summary of the operations a user can carry out with the overview plus detail pane:
+
+- `Zoom a chart incrementally`: Use the mouse wheel or the zoom in/out buttons of the overview plus detail pane.
+- `Zoom a chart to a specific level`: Use the zoom slider of the overview plus detail pane.
+- `Reset a chart to 100% zoom level`: Use the zoom reset button of the overview plus detail pane.
+- `Pan chart in all directions`: Click and drag the window rectangle inside of the preview plot area on the overview plus detail pane.
+- `Pan to a specific region of the chart`: Clicking outside of the window rectangle inside of the preview area will move the window rectangle to that area.
+- `Change mouse drag interaction in the chart`: Use the cursor button on the overview plus detail pane. This will switch between panning and drag zooming with the mouse.
+
+The following code snippet demonstrates how to enable the overview plus detail pane:
+
+```html
+// TODO
+```
+
+```tsx
+<IgrDataChart dataSource={this.data}
+    width="100%"
+    height="400px"
+    overviewPlusDetailPaneVisibility="Visible">
+</IgrDataChart>
+``` -->
+
+## Data Chart Navigation through Code
+
+The $PlatformShort$ data chart component provides several navigation properties that are updated each time a zoom or pan operation happens in the data chart. You can also set each of these properties to zoom or pan the data chart programmatically. The following is a list of these properties:
+
+- `WindowPositionHorizontal`: A numeric value describing the X portion of the content view rectangle displayed by the data chart.
+- `WindowPositionVertical`: A numeric value describing the Y portion of the content view rectangle displayed by the data chart.
+- `WindowRect`: An `IgRect` object representing a rectangle that represents the portion of the chart that is currently in view. For example, a `WindowRect` of "0, 0, 1, 1" would be the entirety of the data chart.
```

Comment:
Change IgRect to Rect. Always use WPF name of properties/classes when creating API links

Reply:
pending